frontend/src/scenes/billing/UnsubscribeSurveyModal.tsx

```diff
@@ -34,13 +34,19 @@ export const UnsubscribeSurveyModal = ({ product }: { product: BillingProductV2T
                             <Link
                                 to="https://posthog.com/docs/billing/estimating-usage-costs#how-to-reduce-your-posthog-costs"
                                 target="_blank"
+                                onClick={() => {
+                                    reportSurveyDismissed(surveyID)
+                                }}
                             >
                                 reduce your bill
                             </Link>
                             {`${product.type !== 'session_replay' ? ' or ' : ', '}`}
                             <Link
                                 to="mailto:sales@posthog.com?subject=Help%20reducing%20PostHog%20bill"
                                 target="_blank"
+                                onClick={() => {
+                                    reportSurveyDismissed(surveyID)
```

Comment:
Closing the survey every time we click on a link will make it easier to do funnel analysis because we'll force the user to start from the "unsubscription survey shown" step.